src/main/java/spoon/compiler/SpoonFile.java

```diff
@@ -41,4 +45,25 @@
 	 * @return
 	 */
 	boolean isActualFile();
+
+	/**
+	 * Gets the file content as a char array, considering encoding or encoding
+	 * provider.
+	 */
+	default char[] getContentChars(Environment env) {
+		byte[] bytes;
+		try {
+			InputStream contentStream = getContent();
+			bytes = new byte[contentStream.available()];
```

Comment:
There is no way how to get size of the input stream.
Use `ByteArrayOutputStream`, then IOUtils.copy to copy input to output and then `ByteArrayOutputStream#toByteArray`